tests/test_connector.py

```diff
@@ -0,0 +1,66 @@
+from logging import getLogger
+from unittest import mock
+
+from pytest import fixture, fail
+
+import code42_connector
+from phantom.action_result import ActionResult
+import phantom.app as phantom
+import py42.sdk
+from requests import HTTPError
+
+
+logger = getLogger(name=__name__)
+
+
+NULL_VALUE = object()
+
+
+@fixture
+def py42_sdk(mocker):
+    return mocker.MagicMock(spec=py42.sdk)
+
+
+@fixture
+def py42_client(mocker):
+    return mocker.MagicMock(spec=py42.sdk.SDKClient)
+
+
+@fixture
+def connector(request, py42_client):
+    connector = code42_connector.Code42Connector()
+    connector._client = py42_client
+    return connector
+
+
+def test_handle_connectivity(mocker, connector):
+    param = {}
+    mocker.patch('phantom.base_connector.BaseConnector.add_action_result', return_value=ActionResult(dict(param)))
+    set_status = mocker.patch('phantom.action_result.ActionResult.set_status')
+
+    connector._handle_test_connectivity(param)
```

Comment:
It is a little odd that we are testing a protected method.. Does `handle_test_connectivity()` need to be a protected method? I am guessing it is called outside the connector class, no?